### DISCUSSION TITLE
github pages<title>

### DISCUSSION THREAD

**Discussion by Achuth568** [community/community#47977] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

first time my github pages worked well,after second commit i want to update my github page.then i choose unpublish this site and i am trying to make a github page.but its not working.i want help from your side.

**Comment by rilysh** [comment]
@Achuth568 Kindly would you share which repository the website belongs?

Also, you have to wait a little until the site published, and the time depends on what site builder you're using. You also, don't have to unpublished your site, just update the files and GitHub actions will respawn the process as the same when the site was first published.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Achuth568, we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
